tools/distrib/buildifier_format_code_strict.sh

```diff
@@ -17,4 +17,4 @@
 dir=$(dirname "${0}")
 buildifier_format_script="${dir}/buildifier_format_code.sh"
 
-${buildifier_format_script} -lint warn
+${buildifier_format_script} -lint fix
```

Comment:
Should we just add the `-lint fix` to the `buildifier_format_code.sh` script, and remove this file?

Reply:
I've found it useful to have a script that proxies flags to buildifier verbatim. The problem with adding `-lint fix` unconditionally is that there are certain other flags that cannot be used with it.